tools/chef/roles/db-mysql56-server.json

```diff
@@ -17,12 +33,14 @@
         "packages": [
           "Percona-Server-devel-56",
           "Percona-Server-server-56"
-        ]
+        ],
+        "service_name": "mysql"
```

Comment:
Put it back to the default again (which the mysql v4.1.2 cookbook will make `mysqld` as expected).